Corzbank.Helpers/MappingProfile.cs

```diff
@@ -23,6 +23,8 @@ public MappingProfile()
 
             CreateMap<DepositModel, Deposit>().ReverseMap();
 
+            CreateMap<Deposit, DepositDTO>().ReverseMap();
```

Comment:
Are there any difference between DepositModel and DepositDTO?

Reply:
sure, depositModel have 3 prop, depositDTO near 10

Reply:
Do properties from the deposit model exist in DepositDTO? If so, you should inherit one another

Reply:
resolved